README.md

```diff
@@ -75,6 +75,50 @@ concord-bft is available under the [Apache 2 license](LICENSE).
 
 Build (Ubuntu Linux 18.04)
 ----
+
+### Install Conan Package Manager
+
+Install Python 3
+
+    sudo apt-get update
+    sudo apt-get install python3
+
+After the installation, please verify the version by running:
+
+    python3 --version
+
+It should be
+
+    Python 3.x.x
+    
+Install with pip
+    
+    sudo apt install python3-pip
+
+Install Conan:
+
+    pip install conan
+    source ~/.profile
```

Comment:
Same question as above. It's necessary only if there environment variables defined in .profile

Reply:
The same answer as above.
When Conan is installed with pip install, usually a new directory is created for it. However, the directory is not appended automatically to the PATH and the Conan commands do not work. This can usually be solved restarting the session of the terminal or running "source ~/.profile"